src/openms/source/FORMAT/DATAACCESS/SiriusFragmentAnnotation.cpp

```diff
@@ -135,26 +265,28 @@ namespace OpenMS
       }
 
       // use first file in folder (rank 1)
-      dir.setFilter(QDir::Files | QDir::Hidden | QDir::NoSymLinks);
-      QFileInfoList list = dir.entryInfoList();
-      QFileInfo firstfile = list[0]; 
+      String filename = rank_filename.at(1); // rank 1
```

Comment:
what if folder empty?

Reply:
Log_Debug output see line 332

Reply:
Yes, you are correct. I moved the functions now, that the directory is checked beforehand. Since if a spectra/decoy directory is available, then the specific files are there as well. 

```
      std::map< Size, String > rank_filename = SiriusFragmentAnnotation::extractCompoundRankingAndFilename_(path_to_sirius_workspace);
      std::map< Size, double > rank_score = SiriusFragmentAnnotation::extractCompoundRankingAndScore_(path_to_sirius_workspace);
```